docs/book/reflection-abstract-factory.md

```diff
@@ -6,10 +6,40 @@ Writing a factory class for each and every service that has dependencies
 can be tedious, particularly in early development as you are still sorting
 out dependencies.
 
-laminas-servicemanager ships with `Laminas\ServiceManager\AbstractFactory\ReflectionBasedAbstractFactory`,
-which provides a reflection-based approach to instantiation, resolving
-constructor dependencies to the relevant services. The factory may be used as
-either an abstract factory, or mapped to specific service names as a factory:
+To alleviate this issue during development, laminas-servicemanager ships with the [ReflectionBasedAbstractFactory](https://github.com/laminas/laminas-servicemanager/blob/3.12.x/src/AbstractFactory/ReflectionBasedAbstractFactory.php),
+which provides a [reflection-based approach](https://www.php.net/manual/en/intro.reflection.php) to instantiation, 
```

Comment:
@settermjd 
You have forgotten your own rule here: One Sentence per-line Method!

Reply:
I tried to stick with what was there. But if it doesn't break the writing convention to change it, then I'm all for changing it to one sentence per-line. Less time to maintain.

Reply:
That's now implemented in #135.